Library/ViewModels/PledgeViewModel.swift

```diff
@@ -39,6 +39,7 @@ public typealias PKPaymentData = (displayName: String, network: String, transact
 public struct PledgeViewData: Equatable {
   public let project: Project
   public let rewards: [Reward]
+  public let selectedShippingRule: ShippingRule?
```

Comment:
I am adding this so that the ShippingRule will be passed along to Confirm Details (late pledge flow) or PledgeViewController (crowdfunding flow) and not break any existing behavior. 

This is why you see updates in the other view models. They get configured with this PledgeViewData struct.